uuu/uuu.cpp

```diff
@@ -657,7 +670,7 @@ int progress(uuu_notify nt, void *p)
 		else
 		{
 			string_ex str;
-			str.format("\rSuccess %d    Failure %d    ", g_overall_okay, g_overall_failure);
+			str.format("\rSuccess %d    Failure %d             ", g_overall_okay, g_overall_failure);
 
```

Comment:
what's change in this line?

Reply:
It is a static resize of this specific string to ensure that it will erase `Wait for Known USB Device Appear...` line.